test/todo-protractor-spec.js

```diff
@@ -0,0 +1,8 @@
+// following Protractor's tutorial
+
+describe('todo homepage', function() {
+
+  it('should initialize with a form and no tasks listed', function() {
+    browser.get('http://localhost:3000')
+  });
+});
```

Comment:
Not actually testing anything here - maybe start by expecting the title to be correct?